sql

```diff
@@ -0,0 +1,46 @@
+create table restoraunt_info(id serial,
+	                         r_name varchar(200) not null,
```

Comment:
change to name it is not required to add prefix r_

Reply:
Fixed